### PR TITLE
ci: update workflows to integrate dubbd-iac-aws

### PR DESCRIPTION
**Pull request by MxNxPx**

### CODE REVIEW

**Review by mikevanhemert — DISMISSED** [review]
Has the IAC (or documentation of it) been placed anywhere else to make the transition smooth for downstream consumers?

Love this incremental step!

**Review by justin-o12 — APPROVED** [review]
Just had one clarifying comment.  Otherwise, LGTM!

File: .github/workflows/dubbd-iac-aws/zarf.yaml [hunk 1]
```diff
@@ -192,21 +161,14 @@ components:
...
-            dir: run/velero
-          - cmd: sleep 15 #time to review
           - cmd: ../terraform apply -auto-approve
             dir: run/velero
       onRemove:
         before:
           - cmd: |
               if [ -d "run/velero" ]; then
                 cd run/velero
-                if [ "$(../terraform output force_destroy)" = true ]; then
```
[the remark this answers is not in the document]
Comment (MxNxPx) [reply .github/workflows/dubbd-iac-aws/zarf.yaml:205]: hi @justin-o12 - good catch, that is correct. as this is the intermediate step to no longer packaging IAC as part of DUBBD, the IAC would only be run in CI, so expecting it to always be ephemeral.

**Comment by MxNxPx** [comment]
> Has the IAC (or documentation of it) been placed anywhere else to make the transition smooth for downstream consumers?
> 
> Love this incremental step!

Great question @mikevanhemert !  as part of _this_ PR, there is no documentation around the IAC move.  but based on your feedback, i plan to add some documentation to the PR that will follow this one and implements the removal of the IAC from dubbd-aws.

File: .github/workflows/test-dubbd-aws-upgrade.yaml [hunk 2]
```diff
@@ -71,33 +90,83 @@ jobs:
...
+        id: deploy-ci-dubbd-iac-aws-oci
         env:
           SHA: ${{ github.sha }}
         run: |
+          echo ${{ steps.get_cluster_name.outputs.eks_cluster_name }}
+
+          # deploys the IAC needed for dubbd-aws
           zarf package deploy \
-            oci://ghcr.io/defenseunicorns/packages/dubbd-aws:${{ steps.latest-package-version.outputs.latest-package-version }} \
+            oci://ghcr.io/defenseunicorns/packages/dubbd-iac-aws:${{ steps.latest-package-version.outputs.latest-package-version }} \
```
Comment (mjnagel) [inline .github/workflows/test-dubbd-aws-upgrade.yaml:102]:
```suggestion
            oci://ghcr.io/defenseunicorns/packages/ci-dubbd-iac-aws:${{ steps.latest-package-version.outputs.latest-package-version }} \
```

**Review by mjnagel — APPROVED** [review]
:shipit:

File: .github/workflows/test-dubbd-aws-upgrade.yaml (hunk 2 above)
Comment (MxNxPx) [reply .github/workflows/test-dubbd-aws-upgrade.yaml]: Good eye @mjnagel , this has now been fixed.

**Pull request merged**